### PR TITLE
WIP: Fix bug no 93

### PR DESCRIPTION
**Pull request by thaiphanvan0779**
Fix bug:
Influxdb FDW fails WHERE string comparison other than "=", "!=" operator

### CODE REVIEW

**Comment by lamduongngoc** [comment]
please make commit comment in the following format

Issue title
* Redmine ticket url
* Root cause:
* Solution

**Comment by lamduongngoc** [comment]
Please fix this warning

influxdb_fdw.c: In function ‘influxdbIterateForeignScan’:
influxdb_fdw.c:1005:23: warning: ‘result’ may be used uninitialized in this function [-Wmaybe-uninitialized]
     InfluxDBFreeResult((InfluxDBResult *) result);
                       ^

**Comment by thaiphanvan0779** [comment]
> Please fix this warning
> 
> influxdb_fdw.c: In function ‘influxdbIterateForeignScan’:
> influxdb_fdw.c:1005:23: warning: ‘result’ may be used uninitialized in this function [-Wmaybe-uninitialized]
> InfluxDBFreeResult((InfluxDBResult *) result);
> ^

I fixed warning, please confirmimg

File: deparse.c [hunk 1]
```diff
@@ -476,6 +476,59 @@ foreign_expr_walker(Node *node,
...
+							case INT8OID:
+							case OIDOID:
+							case FLOAT4OID:
+							case FLOAT8OID:
+							case NUMERICOID:
+							case BITOID:
+							case VARBITOID:
+							case BOOLOID:
+							case BYTEAOID:
+							case TIMESTAMPTZOID:
```
Comment (lamduongngoc) [inline deparse.c:518]: I see in influxdb_query.c, line 176, 177. These types are considered as string.
Could you check string comparison with these types?

**Pull request closed without merging**